README.md

```diff
@@ -91,23 +91,23 @@ extern "C" {
 ```
 (from [imports.rs](https://github.com/confio/cosmwasm/blob/master/src/imports.rs#L12-L17))
 
-You could actually implement a Web Assembly module in any language, 
+You could actually implement a Web Assembly module in any language,
```

Comment:
I think the above list is also outdated.
As you are fixing this README with the purpose of clarifying wasm FFI, it would be nice to clean up a bit more.

I know query is added as an export (from the contract) and c_human_address and c_canonical_address have been added as an import (from runtime to contract).

Reply:
README updated